core/capabilities/triggers/logevent/logeventcap/event_trigger-schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/smartcontractkit/chainlink/core/capabilities/triggers/logevent/log-event-trigger",
+    "$defs": {
+        "head": {
+            "type": "object",
+            "properties": {
+                "Height": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "Hash": {
+                    "type": "string",
+                    "minLength": 42
```

Comment:
Do all blockchains have a min 42 characters for their hash?

Reply:
Changed this to minimum 1